_g3/tests/test-vk-wp-oembed-blog-card.php

```
@@ -20,9 +20,58 @@
+	 *
+	 * `pre_http_request` フィルタは false 以外を返すと実際の HTTP 通信を行わずに
+	 * その値を結果として使う仕様。
+	 * vektor-inc.co.jp ドメインのリクエストのみ `WP_Error` を返して失敗扱いとし、
+	 * YouTube oEmbed や内部リンクなどはそのまま通すために `$pre` を返す。
+	 *
+	 * @param false|array|WP_Error $pre  既存の pre_http_request の戻り値（通常 false）
+	 * @param array                $args HTTP リクエストの引数
+	 * @param string               $url  リクエスト先 URL
+	 * @return false|array|WP_Error vektor-inc.co.jp なら WP_Error、それ以外は $pre をそのまま返す
+	 */
+	public function mock_http_fail( $pre, $args, $url ) {
```

PHPMD (2.15.0)

[warning] Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)